conf/ocsci/deploy_arbiter.yaml

```diff
@@ -0,0 +1,11 @@
+---
+DEPLOYMENT:
+  arbiter_deployment: True
+  arbiter_autodetect: False
+  ocs_operator_nodes_to_label: 4
+  # below conf is based on conf/deployment/aws/ipi_3az_rhcos_3m_4w.yaml
+  arbiter_zone: 'us-east-2c'
+  worker_zones: ['us-east-2a', 'us-east-2b']
```

Comment:
i guess this need's to be generated dynamically because this will not work for VMware deploymeny

Reply:
There are 2 ways to deploy an arbiter, one is static and another one is dynamic.
[1] In static, you need to feed zones in conf file `conf/ocsci/deploy_arbiter.yaml`
[2] In dynamic, zones will get autodetect `conf/ocsci/deploy_arbiter_autodetect.yaml`

Note: For both deployments, you need to use different conf files.